config.default.yaml

```diff
@@ -93,6 +92,12 @@ electricity:
   co2base: 1.487e9  # European default, adjustment to Africa necessary
   agg_p_nom_limits: data/agg_p_nom_minmax.csv
 
+  operational_reserve: # like https://genxproject.github.io/GenX/dev/core/#Reserves
+    activate: false
+    epsilon_load: 0.02 # share of total load
+    epsilon_vres: 0.02 # share of total renewable supply
+    contingency: 0 # fixed capacity in MW
```

Comment:
Ok for the time being, in the future, contingency may be calculated as a fraction of the installed capacity plus a security factor

Reply:
Let's see if it is necessary in future. People can add it on demand.